FAQ.md

```diff
@@ -43,7 +43,7 @@
 *   *What is the "Windows installer"?*
 
     We have built a small installation helper for Windows
-    that installs `nano` and `sqlite`, adds R to the path, and so on.
+    that installs nano and `sqlite`, adds R to the path, and so on.
```

Comment:
On Tue, Apr 14, 2015 at 12:23:04PM -0700, Raniere Silva wrote:
> > -    that installs `nano` and `sqlite`, adds R to the path, and so on.
> > +    that installs nano and `sqlite`, adds R to the path, and so on.
> 
> `sqlite` -> SQLite

Done thanks.  I'd forgotten I was editing the FAQ too.